packages/devtools/web/styles_dark.css

```diff
@@ -57,3 +57,18 @@ nav.menu .menu-item {
     color: wheat;
     font-weight: bold;
 }
+
+/* Tweak how buttons look in the dark theme to not be so dark */
+.btn:not(.btn-primary), .btn:not(.btn-primary)[disabled] {
+    background-image: linear-gradient(-180deg, #3f3f3f 0%, #3f3f3f 90%);
+}
+
+.btn.selected:not(.btn-primary), .btn.selected:not(.btn-primary)[disabled] {
+    background-image: linear-gradient(-180deg, #1f1f1f 0%, #1f1f1f 90%);
+    box-shadow: none;
+}
+
+.btn.invert-dark .flutter-icon {
```

Comment:
remove this CSS and push the logic into html_icons

Reply:
Done!